system/src/invoke_signed.rs

```diff
@@ -0,0 +1,26 @@
+use solana_nostd_entrypoint::{AccountInfoC, InstructionC};
+
+use crate::ProgramResult;
+
+pub fn invoke_signed(
```

Comment:
Calling this method invoke signed is a bit misleading

Reply:
how so? can you elaborate more ser

Reply:
https://github.com/Aursen/nostdinfo-programs/blob/274906c4c3d18f6fa9e3fe8d286106c395ed45b4/crates/invoke/src/lib.rs#L18

For example, in the verified call, we check whether the metas accounts agree with the accounts in the instruction. Since .to_metas-C is account-based, I don't know if it's accurate in this context @cavemanloverboy any ideas?